src/coreclr/src/tools/r2rdump/R2RDump.csproj

```diff
@@ -15,15 +15,16 @@
     <Platforms>AnyCPU;x64</Platforms>
   </PropertyGroup>
 
+  <Import Project="$(RepositoryEngineeringDir)coredistools.targets" />
   <Import Project="$(RepositoryEngineeringDir)DiaSymReaderNative.targets" />
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.CoreDisTools">
-      <Version>1.0.1-prerelease-00005</Version>
-    </PackageReference>
     <PackageReference Include="System.CommandLine">
       <Version>$(SystemCommandLineVersion)</Version>
     </PackageReference>
     <ProjectReference Include="..\aot\ILCompiler.Reflection.ReadyToRun\ILCompiler.Reflection.ReadyToRun.csproj" />
+    <Content Include="$(CoreDisToolsLibrary)" Condition="Exists('$(CoreDisToolsLibrary)')">
```

Comment:
What's the purpose of the <code>Exists</code> check, isn't that rather hiding a build inconsistency? I was under the impression that thanks to your work we now have <code>coredistools</code> available on all supported platforms.

Reply:
There are no coredistools library for linux-musl and some unsupported platforms (such as linux-x86). Since R2RDump is a .NET Core app, in theory, it can be built targeting these platforms. The check is to prevent the build from failure (but, obviously, the disassembler would not work).

Reply:
I see, thanks for clarifying!